test/Altinn.Profile.Tests/Testdata/UserProfile/4c3b4909-eb17-45d5-bde1-256e065e196a.json

```diff
@@ -0,0 +1,48 @@
+{
+  "UserId": 20000006,
+  "UserUUID": "4c3b4909-eb17-45d5-bde1-256e065e196a",
+  "UserType": 1,
+  "UserName": "",
+  "ExternalIdentity": "",
+  "PhoneNumber": null,
+  "Email": null,
+  "PartyId": 50002114,
+  "Party": {
+    "PartyTypeName": 1,
+    "SSN": "01025161013",
+    "OrgNumber": "",
+    "Person": {
+      "SSN": "01025161013",
+      "Name": "ELENA FJÆR",
```

Comment:
Test is failing because of this.

Reply:
Good test :)

Reply:
Could not find out why the Æ was encoded wrong on build machine when it was stored directly in source code. It was working on Dev machine so ended up just ignoring to test this and removed all Æs from the test.